tests/cppunit/ir_pass_test.cc

```diff
@@ -111,18 +111,35 @@ TEST(IRPassTest, Manager) {
             "select * from a where (and x <= 1, y >= 2, z != 3)");
 }
 
+TEST(IRPassTest, SortByWithLimitToKnnExpr) {
+  SortByWithLimitToKnnExpr tsbtke;
+
+  ASSERT_EQ(tsbtke.Transform(*Parse("select a from b order by embedding <-> [3.6] limit 5"))->Dump(),
+            "select a from b where KNN k=5, embedding <-> [3.600000] limit 0, 5");
+  ASSERT_EQ(tsbtke.Transform(*Parse("select a from b where false order by embedding <-> [3,1,2] limit 5"))->Dump(),
+            "select a from b where false sortby embedding <-> [3.000000, 1.000000, 2.000000] limit 0, 5");
```

Comment:
I'm wondering if this query plan is valid and have a correct execution result 🤔

Reply:
For `"select a from b where false order by embedding <-> [3,1,2] limit 5"`, this will be set as `noop` in `LowerToPlan`.